src/main/java/com/stripe/BranchCoverageUtil.java

```diff
@@ -0,0 +1,67 @@
+package com.stripe;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.io.File;
+
+public class BranchCoverageUtil {
+  public static void writeDefault() {
+    String filePath = "branchCoverage.txt";
+
+    File file = new File(filePath);
+    try {
+      file.createNewFile();
+    } catch (IOException e) {
+      Logger.getLogger(BranchCoverageUtil.class.getName()).log(Level.SEVERE, "Failed to create to file: " + filePath, e);
```

Comment:
You could create a static variable and then use that instead of getting the logger every time you log or sth idk
```java
static private Logger logger = Logger.getLogger(BranchCoverageUtil.class.getName());
```

Reply:
Smort will fix mr boss man